main.rb

```diff
@@ -1,34 +1,29 @@
 require 'discordrb'
-require 'pry'
-require 'hocon'
-require 'fuzzystringmatch'
+
+# services
 require_relative 'services/discord_message_sender'
-require_relative 'services/building_service'
-require_relative 'services/latex_service'
+
+# config module
+require_relative './config'
+
+# modules
+require_relative 'modules/event_roles'
+require_relative 'modules/purge'
+require_relative 'modules/equation'
+require_relative 'modules/year'
+require_relative 'modules/where_is'
 
 class Main
-  CONFIG = Hocon.load("config.conf")
-  IMAGE_DIRECTORY_URL = CONFIG["urls"]["image_directory_url"]
-  LATEX_DIRECTORY_RELATIVE_PATH = "tmp"
-  BOT_USER_ID = CONFIG["bot_user_id"]
-  EXCLUDE_ROLES = [
-    "Bot",
-    "Admin",
-    "Moderator",
-    "CSS President",
-    "CSS Board Executive",
-    "CSS Board Head",
-    "CSS Board Member",
-  ]
-  
+ 
+  # startup sequence
   bot = Discordrb::Commands::CommandBot.new(
-    token: CONFIG["api_token"],
-    client_id: CONFIG["api_client_id"],
-    prefix: '~',
+    token: Config::CONFIG["api_token"],
+    client_id: Config::CONFIG["api_client_id"],
```

Comment:
Instead of loading them directly from Config::CONFIG you could just define them like you did here:
https://github.com/UWindsorCSS/uwindsor-discord-bot/blob/a3538be1e824942a934d47bf5f1d20aa01162d1d/config.rb#L7-L9
Any reference them here like `Config::API_TOKEN`

Reply:
thats a good point, I should stick with referencing variables because it leads to more flexibility.